### ISSUE TITLE
Build error for OCaml-LGPL-Linking-Exception

### ISSUE THREAD

**Issue by swinslow** [spdx/license-list-XML#705]
When running `make validate-canonical-match` on the current master branch, commit b49e272, I'm getting the following error message (reformatted for clarity)

```
Test for exception OCaml-LGPL-Linking-Exception failed: 

Test for exception ID OCaml-LGPL-Linking-Exception failed due to difference found Normal text of license does not match starting at line #1 column #3 "the" when comparing to template text "As a special exception to the GNU ".  

Last optional text was not found due to the optional difference:
Normal text of license does not match starting at line #1 column #3 "the" when comparing to template text " OCaml LGPL Linking Exception"
```

It looks like this is happening because of extra text at the top of the test file for OCaml-LGPL-Linking-Exception. I'm preparing a PR to fix this.

**Comment by swinslow** [comment]
Addressed in #706, along with:

- there was also previously a build error due to a missing quote mark + space in the markup
- I've modified the capitalization of the licenseId, so that it will match the filenames (and the capitalization that's used for other SPDX exceptions)

**Comment by jlovejoy** [comment]
now resolved and PR has been merged
